components/home/OurServices.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <section>
+    <v-container>
+      <app-section-descriptor
+        title="Services"
+        heading="Core Offerings"
+        description="Users care about Privacy and Security, set your app apart with the most developer friendly data storage and privacy platform. Takes minutes to integrate."
+        style="max-width: 64rem"
+      />
+      <div class="overview-features">
+        <v-stack align="center" sm-align="start" gap="20vw" sm-gap="1rem">
```

Comment:
There's a repeated markup here. See if you want to separate out the data in the script and iterate over it in the markup.

Reply:
Will need to do this separately, currently we have following design

```
Content    Icon
Icon          Content
Content    Icon
```

Our VStack component is not properly capable of handling situations like direction row reverse. Hence I repeated the markup